libs/coin-evm/src/__tests__/unit/logic.unit.test.ts

```diff
@@ -681,5 +683,37 @@ describe("EVM Family", () => {
         expect(padHexString("01")).toEqual("01");
       });
     });
+
+    describe("safeEncodeEIP55", () => {
+      it("Should return encoded address if valid address", () => {
+        const address = "0x9aa99c23f67c81701c772b106b4f83f6e858dd2e";
+        const encodedAddress = safeEncodeEIP55(address);
+        expect(encodedAddress).toBe(eip55.encode(address));
```

Comment:
By using `eip55.encode`, you are using and relying on implementation details of the function you are actually testing. I would rather write directly the expected output.
This makes the test more resilient (and actually agnostic) to implementation updates of the underlying function.

```suggestion
        expect(encodedAddress).toBe("0x9AA99C23F67c81701C772B106b4F83f6e858dd2E");
```